### DISCUSSION TITLE
CLOSED - Prometheus TSDB snapshots from multiple systems to single prometheus server

### DISCUSSION THREAD

**Discussion by rajdevworks** [prometheus-operator/prometheus-operator#4879] [Q&A] [answered]
Hello,

I have N systems in my cluster and collect a prometheus TSDB snapshot as required. The cluster is in isolated environment but the snapshot can be extracted to another environment where prometheus/grafana are running.

I'm able to visualize single snapshot by placing the data under /data directory and running prometheus.
If I have multiple TSDB snapshots from different systems, how can it be stored on Prometheus server such that they can be queried individually?
How can I apply external labels which can be translated to Grafana variables to filter on the Grafana dashboard?

**Comment by simonpasquier** [comment, marked as answer]
:wave: this sounds more like a question for the Prometheus community.

**Reply by rajdevworks** [reply]
Thank you @simonpasquier . I will open a thread in Prometheus community.

**Reply by rajdevworks** [reply]
For reference - https://github.com/prometheus/prometheus/discussions/10940
